site/en/community/contribute/community.md

```diff
@@ -18,95 +32,37 @@ The TensorFlow community has a number of formal and informal ways of keeping in
 
 The primary communication about work on TensorFlow happens in the [TensorFlow repositories on GitHub](https://github.com/tensorflow). This is the place to discuss bugs, new features, and in-progress work.
 
-<!--
-### Forums
--->
-
 ### Mailing lists
 
-Mailing lists are reserved for announcements and contributor conversation. They are not intended to provide technical support.
-
-#### General TensorFlow lists
+Most communication happens on the TensorFlow Forum, however the following mailing lists are still used for announcements and contributor conversation. They are not intended to provide technical support.
```

Comment:
```suggestion
Most communication happens on the TensorFlow Forum. The following mailing lists are still used for announcements and contributor conversations (note that they are not intended to provide technical support).
```

LMKWYT

@theadactyl @kristenrq

Reply:
Nit: I like breaking the first sentence up, but I would leave the last sentence as its own sentence ("Note that...", as I think the point is more than parenthetical.

Reply:
Done @wolffg